client/app/pages/queries/query.html

```diff
@@ -90,157 +90,56 @@ <h3>
       </div>
 
       <div class="editor__left__schema" ng-if="sourceMode">
-        <schema-browser class="schema-container" schema="schema" tabletogglestring="dataSource.options.toggle_table_string" on-refresh="refreshSchema()"></schema-browser>
+        <schema-browser class="schema-container" schema="schema" tabletogglestring="dataSource.options.toggle_table_string" on-refresh="refreshSchema" editor-paste="editorPaste"></schema-browser>
       </div>
       <div ng-if="!sourceMode" style="flex-grow: 1;">&nbsp;</div>
 
       <div class="query-metadata query-metadata--description">
-        <edit-in-place editable="canEdit" done="saveDescription" editor="textarea" placeholder="Add description" ignore-blanks='false'
-          value="query.description" markdown="true">
+        <edit-in-place
+          class="edit-in-place"
+          editable="canEdit"
+          on-done="saveDescription"
+          editor="'textarea'"
+          placeholder-text="'Add description'"
```

Comment:
As John noted (and looking at the modified diff) the embedded quotes seem wrong.

Reply:
Angular, man. :roll_eyes: